### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by anjor**

### CODE REVIEW

File: examples/add-buffer.py [hunk 1]
```diff
@@ -1,4 +1,9 @@
...
 
-buffer = "The contents of my file to upload"
-print(add_string(buffer, "testfile.txt"))
+def example_add_string():
+    buffer = "The contents of my file to upload"
+    print(add_string(buffer, "testfile.txt"))
+
+
+if __name__ == '__main__':
+    test_add_string()
```
Comment (snissn) [inline examples/add-buffer.py:9]: function name and function call aren't consistent!

Comment (gmelodie) [inline examples/add-buffer.py:9]: Don't create a function for this. It confuses the example. At max do
```python
if __name__ == '__main__':
    buffer = "The contents of my file to upload"
    print(add_string(buffer, "testfile.txt"))
```

File: examples/add-dir-to-collection.py [hunk 2]
```diff
@@ -1,45 +1,13 @@
...
-        random_content = ''.join(random.choice(string.printable) for _ in range(10))
-        f.write(random_content)
-        f.close()
+from utils import cleanup_test_data_directory, setup_test_data_directory
 
-# add all the files under /tmp/test-adddir
-# responses = content_add('/tmp/test-adddir')
+if __name__ == '__main__':
+    directory = 'test-addir'
+    setup_test_data_directory(directory)
```
Comment (gmelodie) [inline examples/add-dir-to-collection.py:8]: `create_tmp_directory` is a better name

File: examples/add-dir-to-collection.py [hunk 3]
```diff
@@ -1,45 +1,13 @@
...
-# on estuary using collections
-responses, collection = content_add('/tmp/test-adddir', create_collection=True)
-print(collection_list_content(collection.uuid))
+    responses, collection = content_add('/tmp/test-adddir', create_collection=True)
+    print(collection_list_content(collection.uuid))
 
-# you can even list contents recursively!!
-# not sure what we want recursive vs non recursive to be
-# print(json.dumps(collection_list_content(collection["uuid"], "/", recursive=True), indent=4))
+    cleanup_test_data_directory(directory)
```
Comment (gmelodie) [inline examples/add-dir-to-collection.py:13]: `remove_tmp_directory`

File: examples/add-files-to-collection.py [hunk 4]
```diff
@@ -1,34 +1,22 @@
...
-
-    return content_id
+    resp = content_add(test_file_name)
+    return resp.estuary_id
 
 
 if __name__ == '__main__':
     file_name = 'a.csv'
-    setup_test_data(file_name)
+    setup_test_data_file(file_name)
```
Comment (gmelodie) [inline examples/add-files-to-collection.py:19]:
```suggestion
    create_tmp_file(file_name)
```

File: examples/create-collection.py [hunk 5]
```diff
@@ -2,7 +2,13 @@
 
 from pestuary.collections import collection_create
 
-name = "Test"
-description = "Helpful description here"
-response = collection_create(name, description)
-print(response)
+
+def example_collection_create():
```
Comment (gmelodie) [inline examples/create-collection.py:6]: Same as above. Putting this code in a function does not make it more readable and it confuses the example

File: examples/list-collection.py [hunk 6]
```diff
@@ -1,3 +1,8 @@
...
-response = collection_list()
-print(response)
+from pestuary.collections import collection_list
+
+def example_collection_list():
+    response = collection_list()
+    print(response)
+
+if __name__ == '__main__':
+    example_collection_list()
```
Comment (gmelodie) [inline examples/list-collection.py:8]: Same as above. Putting this code in a function does not make it more readable and it confuses the example

**Review by gmelodie — CHANGES_REQUESTED** [review]

**Pull request merged**